exporters/mongodb3/mongodb3.prometheus.json.tmpl

```diff
@@ -8,7 +8,7 @@
          {
             "name": "nri-prometheus",
             {{- if .env.interval }}
-            "interval": "{{.env.interval}}",
+            "interval": "{{.env.NRI_CONFIG_INTERVAL}}",
```

Comment:
We would be using `.env.NRI_CONFIG_INTERVAL` only if `.env.interval`, wouldn't we?